src/SceneKit/SCNVector4.cs

```diff
@@ -1103,6 +1126,46 @@ public bool Equals(SCNVector4 other)
 		return new Vector4 ((float)source.X, (float)source.Y, (float)source.Z, (float)source.W);
 	}
 
-		
+#if NET
+        /// <summary>
+        /// Returns an approximation of the inverse square root of left number.
+        /// </summary>
+        /// <param name="x">A number.</param>
+        /// <returns>An approximation of the inverse square root of the specified number, with an upper error bound of 0.001</returns>
+        /// <remarks>
+        /// This is an improved implementation of the the method known as Carmack's inverse square root
+        /// which is found in the Quake III source code. This implementation comes from
+        /// http://www.codemaestro.com/reviews/review00000105.html. For the history of this method, see
+        /// http://www.beyond3d.com/content/articles/8/
+        /// </remarks>
+        static float InverseSqrtFast(float x)
```

Comment:
This is the second time InverseSqrtFast shows up, maybe make one of the `internal static` and call it elsewhere?